### PR TITLE
More Kotlin conversions

### PR DESCRIPTION
**Pull request by hannesa2**

### CODE REVIEW

**Comment by niqdev** [comment]
All prs are still in `Expected — Waiting for status to be reported ` 🤷

File: mjpeg-view/src/main/java/com/github/niqdev/mjpeg/Mjpeg.java [hunk 1]
```diff
@@ -21,8 +21,8 @@
 /**
  * A library wrapper for handle mjpeg streams.
  *
- *     <li><a href="https://bitbucket.org/neuralassembly/simplemjpegview">simplemjpegview</a></li>
```
Comment (niqdev) [inline mjpeg-view/src/main/java/com/github/niqdev/mjpeg/Mjpeg.java:24]: there is a `<ul>` missing

**Comment by hannesa2** [comment]
I've no clue from where this comes from 
image

When I do the same pull request on my fork 
https://github.com/hannesa2/ipcam-view/pull/42

There is nothing like this ? 
image

File: mjpeg-view/src/main/java/com/github/niqdev/mjpeg/Mjpeg.java (hunk 1 above)
Comment (hannesa2) [reply mjpeg-view/src/main/java/com/github/niqdev/mjpeg/Mjpeg.java:24]: Lint show me a warning about this. Here I'm the executor and simply deleted it

Comment (niqdev) [reply mjpeg-view/src/main/java/com/github/niqdev/mjpeg/Mjpeg.java:24]: go ahead, whatever you think it's best.. you could just keep the link without the html tbh

Comment (hannesa2) [reply mjpeg-view/src/main/java/com/github/niqdev/mjpeg/Mjpeg.java:24]: It is like it is

**Pull request merged**